recipes/wntr/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "wntr" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/michaelbynum/WNTR/archive/f73f0618e22201c0d15e5d1c62537564fc8840cb.zip
+  sha256: ed3b323ebe7bac1cce88520294327f61327d752f0e5086a28c909896948f799b
+
+build:
+  number: 0
+  skip: true  # [win]
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
```

Comment:
It is unclear from looking at the github repo if this package compiles any code/has c-extensions. It doesn't look like it, but I might have missed something. If there is no code to compile, then you can drop the `build` section.

Reply:
@synapticarbors Thanks for your feedback. The package does compile C extensions. I am actually doing this work on a branch of a fork of the main repo (the work is being done in michaelbynum/WNTR on a branch called hydraulics4), which is probably why you did not see any indication of compiled extensions.